### DISCUSSION TITLE
5G Network Simulation

### DISCUSSION THREAD

**Discussion by abidachiever** [Unipisa/Simu5G#83] [Q&A] [answered]
Hello Team,

Hope you are doing well!

Can you please guide me on where to start to create a basic 5G network in Omnet++. Any link reference link or document will help alot.

Thanks

**Comment by KourosZanbouri** [comment, marked as answer]
Hi 

> Hello Team,
> 
> Hope you are doing well!
> 
> Can you please guide me on where to start to create a basic 5G network in Omnet++. Any link reference link or document will help alot.
> 
> Thanks

Hi,

Although it is an old discussion. try to import the latest version of Simu5G ( you will need Omnet++ and iNet). Then you can find sample configs. For example, you can find a simple 5G MEC under : Simu5G > Simulations > NR > MEC > SingleMecHost
